source/common/common/base64.cc

```diff
@@ -1,8 +1,53 @@
 #include "base64.h"
 
+#include "common/common/assert.h"
+
 static constexpr char CHAR_TABLE[] =
     "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 
+static const unsigned char REVERSE_LOOKUP_TABLE[256] = {
```

Comment:
how was this generated? If you copied this from somewhere document where it was copied from.